oem/ibm/libpldmresponder/inband_code_update.cpp

```diff
@@ -280,12 +280,25 @@ void CodeUpdate::setVersions()
             nextBootSide =
                 (pldmBootSideData.next_boot_side == "Temp" ? Tside : Pside);
         }
+        retrySetVersion = 0;
```

Comment:
we initialized directly in the class right ? why to re-initialize it again ?

Reply:
done, removed